search/fuse/fuse.md

```diff
@@ -13,6 +13,8 @@ You can think of fuse like a simpler version of [lookup](/search/lookup/lookup),
 
 ## Supported Options
 
+* `-maxtracked <arg>`: sets the maximum number of unique keys to track per operation. The query will abort if this value is exceeded. Defaults to 1000000.
+* `-maxsize <arg>`: sets the maximum amount of memory in megabytes to hold when tracking keys. Defaults to 1GB.
```

Comment:
I think we should express the default as MB since that is the unit of measure we are using.

Reply:
Done